docs-site/docs/contribute-code.md

```diff
@@ -18,120 +18,70 @@ A simple way to enforce the Google coding styles is to use the linting and forma
 
 Once the extensions are installed, edit the `settings.json` file.
 
-```json 
+```json
 {
-   "[cpp]": {
-      "editor.defaultFormatter": "xaver.clang-format"
-   },
-   "cpplint.cpplintPath": "path/to/cpplint",
-
-   "editor.formatOnSave": true,
-   "python.formatting.provider": "yapf",
-   "python.linting.enabled": true,
-   "python.linting.lintOnSave": true,
-   "clang-format.language.cpp.style": "google",
-   "python.formatting.yapfArgs" : ["--style" , "{based_on_style: google}" ]
+  "[cpp]": {
+    "editor.defaultFormatter": "xaver.clang-format"
+  },
+  "cpplint.cpplintPath": "path/to/cpplint",
+
+  "editor.formatOnSave": true,
+  "python.formatting.provider": "yapf",
+  "python.linting.enabled": true,
+  "python.linting.lintOnSave": true,
+  "clang-format.language.cpp.style": "google",
+  "python.formatting.yapfArgs": ["--style", "{based_on_style: google}"]
 }
 ```
 
 Depending on your platform, the user settings file is located here:
+
 1. Windows %APPDATA%\Code\User\settings.json
-2. macOS $HOME/Library/Application Support/Code/User/settings.json
-3. Linux $HOME/.config/Code/User/settings.json
+2. macOS \$HOME/Library/Application Support/Code/User/settings.json
```

Comment:
@FeynmanDNA does  the git hook add the `\` automatically?  I removed it manually; but after committing, it came back again.

Reply:
probably this is due to the linting and formatting, maybe you can try wrap these paths in ticks

2. macOS `$HOME/Library/Application Support/Code/User/settings.json`
3. Linux `$HOME/.config/Code/User/settings.json`